harness/determined/cli/_color_json.py

```diff
@@ -0,0 +1,92 @@
+"""
+colorize JSON output.
+test string:
+   echo '{"a": "b", "c": ["1", 2.0, 3.1, 5, "4", {"d": true, "e": null, "f": [1, 2, 3]}]}'
+"""
```

Comment:
delete this.  Maybe write a unit test but don't leave this here.

Reply:
Question for both of you: what's normal / policy for unit tests? How often do we want to see them for functionality like this?

Reply:
yeah, it should have a unit test.  Code should be unit-tested in general.

I think some of these cli-focused features we've been lax on, but that's probably a bug, not a feature.